secret/secret.go

```diff
@@ -64,48 +68,147 @@ func UpdateSecrets(s secretInterface, secrets *v1.Secret, create bool) {
 		if err != nil {
 			logFatal(err)
 		}
-		log.Println("Created `secret`")
+		log.Printf("Created `%s`\n", secrets.Name)
 	} else if util.IsDirty(secrets) {
 		util.MarkAsClean(secrets)
 		_, err := s.Update(secrets)
 		if err != nil {
 			logFatal(err)
 		}
-		log.Println("Updated `secret`")
```

Comment:
This whole branch for `!create` goes away because we will always have to create the new versioned secret (see below).

So the `create` parameter goes away, but also all the code handling the `dirty` flag isn't needed anymore either and should be removed.

Reply:
See previous notes.

Reply:
Will do.